.changeset/happy-baboons-suffer.md

```diff
@@ -0,0 +1,5 @@
+---
+"live-mobile": patch
+---
+
+Bump fastlane to support processing multiple matching builds (especially for nightly)
```

Comment:
is a changeset necessary for this type of change? Internal-facing only, only affects our build process

Reply:
well , i've been asking the same question to myself, i'm little bit mitigated, i totally agree with your point, but at the same time it affect, local, adhoc and ci build process for LLM. so it can be necessary. 
We may need the opinion of @KVNLS  on this point. For me it's not a direct CI based dev and not a functional one.

Reply:
It makes sense to add a changeset as dependencies changed for LLM

Reply:
okay then 👍